src/Xamarin.Android.App/Xamarin.Android.App.Runtime.proj

```diff
@@ -0,0 +1,51 @@
+<!--
```

Comment:
What are the *semantics* of `src/Xamarin.Android.App`?  "Current" semantics is that `src/*` produce `.dll` files, `tools/*` produce `.exe` files, and `build-tools` is stuff used for the build.

This file doesn't "feel like" it's producing a `.dll` output.  Should `src/Xamarin.Android.App` instead be `build-tools/Xamarin.Android.App`?

Reply:
Fair point, I've moved this into build-tools. I was wondering if I should move the SDK projects, but since that directory also contains new props/targets I think it can remain in `src`?

Reply:
It doesn't need to be moved at this time.  (It *should* have gone into build-tools in the first place, but was overlooked.)